File: Data Structures & Algorithms/is-palindrome/submission-0.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s = s.lower().replace(" ", "")
        s = re.sub(r"[^a-z0-9]", "", s)
        n = len(s)
        right = n - 1
        left = 0

        while (left<=right):
            
            if(s[left]!= s[right] ):
                return False
            left += 1
            right -= 1
        return True

        
        

File: Data Structures & Algorithms/binary-tree-inorder-traversal/submission-0.cpp
class Solution {
public:
    vector<int> results;

    void inorder(TreeNode* root) {
        if (root == nullptr) return;
        inorder(root->left);
        results.push_back(root->val);
        inorder(root->right);
    }

    vector<int> inorderTraversal(TreeNode* root) {
        inorder(root);
        return results;
    }
};